dotcom-rendering/src/components/KeyTakeaways.tsx

```diff
@@ -0,0 +1,136 @@
+import { css } from '@emotion/react';
+import type { ArticleFormat } from '@guardian/libs';
+import { headline } from '@guardian/source-foundations';
+import type { EditionId } from '../lib/edition';
+import { RenderArticleElement } from '../lib/renderElement';
+import { palette } from '../palette';
+import type { ServerSideTests, Switches } from '../types/config';
+import type { KeyTakeaway } from '../types/content';
+
+const keyTakeawayStyles = css`
+	padding-top: 8px;
+`;
+
+const headingStyles = css`
+	${headline.xsmall({ fontWeight: 'medium' })};
+	padding: 2px 0px;
+`;
+
+const headingIndexStyles = css`
+	${headline.xsmall({ fontWeight: 'bold' })};
+`;
+
+const headingLineStyles = css`
+	width: 140px;
+	margin: 0px;
+	border: none;
+	border-top: 8px solid ${palette('--heading-line')};
+`;
+
+interface CommonProps {
+	format: ArticleFormat;
+	ajaxUrl: string;
+	host?: string;
+	pageId: string;
+	isAdFreeUser: boolean;
+	isSensitive: boolean;
+	abTests: ServerSideTests;
+	switches: Switches;
+	editionId: EditionId;
+	hideCaption?: boolean;
+	starRating?: number;
+}
+
+interface KeyTakeawaysProps extends CommonProps {
+	keyTakeaways: KeyTakeaway[];
+}
+
+interface KeyTakeawayProps extends CommonProps {
+	keyTakeaway: KeyTakeaway;
+	titleIndex: number;
+}
+
+const KeyTakeawayComponent = ({
+	keyTakeaway,
+	format,
+	ajaxUrl,
+	host,
+	pageId,
+	isAdFreeUser,
+	isSensitive,
+	switches,
+	abTests,
+	editionId,
+	titleIndex,
+	hideCaption,
+	starRating,
+}: KeyTakeawayProps) => {
+	return (
+		<li css={keyTakeawayStyles}>
+			<hr css={headingLineStyles}></hr>
+			<h2 css={headingStyles}>
+				<span css={headingIndexStyles}>{`${titleIndex}. `}</span>
+				{keyTakeaway.title}
+			</h2>
+			{keyTakeaway.body.map((element, index) => (
+				<RenderArticleElement
```

Comment:
This will lead to the [`renderElement`](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/lib/renderElement.tsx#L124) function to be executed and I cannot see handling of the [`KeyTakeawaysBlockElement`](https://github.com/guardian/dotcom-rendering/pull/10846/files#diff-c0f16573f3060f24824d680ce703a4d47d2ccab96bd03413941caf8256c5f32bR322-R324). I guess we don't expect this type of component from upstream yet and this will be added at a later PR?

Reply:
This is a good question. 
We don't expect a key takeaway to ever contain a key takeaway - but possibly we might want to make that explicit -will have a think about this

Reply:
As @alinaboghiu says, CoPro are guaranteeing that these won't ever be nested, i.e. a key takeaway won't ever include another list of key takeaways.

On `renderElement` implementation, we haven't added these yet, will do this in a later PR.